applications/polkadot-contract-wizard.md

```diff
@@ -0,0 +1,227 @@
+- **Full-Time Equivalent (FTE):**  2
+- **Total Costs:** 28,000 USD
+
+### Milestone 1 — UI and Code Generation
+
+- **Estimated duration:** 3 months
+- **FTE:**  2
+- **Costs:** 21,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how a user can generate its own smart contracts. |
+| **0c.** | Testing and Testing Guide | The code will have unit-test coverage to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a docker container with current milestones deliverables to easily run the application. |
+| 0e. | Article | We will publish an article that explains what we have achieved building this project and how this will impact the ecosystem. |
+| 1. | Redesign frontend interface | The functionality to be implemented corresponds to [step 1](#step1) of the Project Details section. |
+| 2. | Develop the interface based on the previous task result | The functionality to be implemented corresponds to [step 2](#step2) of the Project Details section. |
+| 3. | Compose the contract based on the selection | The functionality to be implemented corresponds to [step 3](#step3) of the Project Details section. |
+| 4. | Add syntax highlighting to the displayed smart contract code | The functionality to be implemented corresponds to [step 4](#step4) of the Project Details section. |
+
+### Milestone 2 — Smart Contracts Deployment and Instantiation functionality
+
+- **Estimated duration:** 1 month
+- **FTE:**  2
+- **Costs:** 7,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how a user can instantiate a smart contract. |
+| **0c.** | Testing and Testing Guide | The code will have unit-test coverage to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a docker container with current milestones deliverables to easily run the application. |
+| 0e. | Article | We will publish an article that explains what we have achieved building this project and how this will impact the ecosystem . |
+| 1. | Deploy all possible combinations of smart contracts | The functionality to be implemented corresponds to [step 5](#step5) of the Project Details section. |
+| 2. | Develop Instance functionality | The functionality to be implemented corresponds to [step 6](#step6) of the Project Details section. |
+
+## Future Plans
+
+After the completion of this project, we would love to broaden its scope.
+
+**Multiple Chains**
+- Deploy the set of smart contracts in several parachains that implement the contracts pallet.
```

Comment:
Thanks for the application. Could you remove the "deployment" of the smart contracts from M2? We typically fund the development of the contracts but not the deployment itself, as it isn't specifically useful for others. With this change you might consider reducing the price a bit as well.

Reply:
Hi @keeganquigley, thanks for your feedback. We re-wrote the deployment explanation to avoid missunderstandings. Its important to notice the milestone does not cover the expenses for deploying, it will cover the development for being able to deploy on demand by the user. Let us know if we need to clarify something else.